app/presenters/tree_builder_storage.rb

```diff
@@ -24,8 +24,13 @@ def x_get_tree_roots(count_only, _options)
   end
 
   def x_get_tree_custom_kids(object, count_only, options)
-    return count_only ? 0 : [] if object[:id] != "global"
-    objects = MiqSearch.where(:db => options[:leaf]).visible_to_all
+    objects = MiqSearch.where(:db => options[:leaf])
+    objects = case object[:id]
```

Comment:
it is in app/presenters/tree_builder_storage.rb like that because it needs the same functionality as in https://github.com/ManageIQ/manageiq/blob/master/app/presenters/tree_builder_vms_filter.rb#L28
which works well and it is well understandable what's going on there.